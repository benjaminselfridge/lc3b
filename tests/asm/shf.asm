        0x4500

        ADD     R0      R0      0x1F
        LSHF    R1      R0      4
        RSHFL   R2      R1      3
        RSHFA   R3      R1      4

        TRAP    0x0
        
