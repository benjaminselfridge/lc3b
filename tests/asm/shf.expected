[ RegContains 0 0xFFFF
, RegContains 1 0xFFF0
, RegContains 2 0x1FFE
, RegContains 3 0xFFFF
]